reports/performance/_outliers.qmd

```diff
@@ -368,3 +368,133 @@ qnt_df <- assessment_pin %>%
 qnt_df %>%
   datatable(rownames = FALSE)
 ```
+
+
+## Increase in Housing Value
+
+This section looks at the year over year change in assessed housing values. It separates the data into individual observations, identifying 10 PINs in each neighborhood which grew at the fastest rate as well as the median rate of change for each neighborhood. 
+
+::: panel-tabset
+
+## Map of Largest YOY Changes
+
+This map caps the legend at a 300% to increase legibility. It then plots the 10 largest year over year increases in assessed values by neighborhood. 
+
+```{r _outliers_map_of_largest_change}
+largest_increases <- assessment_pin %>%
+  group_by(meta_nbhd_code) %>%
+  left_join(ccao::nbhd_shp, by = c("meta_nbhd_code" = "town_nbhd")) %>%
+  filter(!is.na(prior_near_yoy_change_pct)) %>%
+  mutate(neighborhod_median_change = median(prior_near_yoy_change_pct * 100), ) %>%
+  arrange(desc(prior_near_yoy_change_pct)) %>%
+  slice(1:10) %>%
+  mutate(
+    prior_near_yoy_change_pct = prior_near_yoy_change_pct * 100,
+    capped_yoy_change = pmin(prior_near_yoy_change_pct, 300)
+  )
+
+
+nbhd_increases <- st_as_sf(largest_increases) %>%
+  distinct(meta_nbhd_code, .keep_all = TRUE)
+
+color_palette <-
+  colorNumeric(palette = c("white", "red"), domain = c(0, 300))
+
+leaflet() %>%
+  addProviderTiles(providers$CartoDB.Positron) %>%
+  addPolygons(
+    data = nbhd_increases,
+    fillColor = "white",
+    weight = 1,
+    color = "black",
+    opacity = 1,
+    fillOpacity = 0.5
+  ) %>%
+  addCircleMarkers(
+    data = largest_increases,
+    ~loc_longitude,
+    ~loc_latitude,
+    color = ~ color_palette(capped_yoy_change),
```

Comment:
**issue:** Even with the cap there's a warning thrown by your color scale:

```
Warning in color_palette(capped_yoy_change): Some values were outside the color
scale and will be treated as NA
```

Reply:
There was one neighborhood with negative rates of housing increase. I fixed the scale to the minimum of the column and it works fine now.